cmd/skopeo/copy.go

```diff
@@ -95,6 +96,7 @@ See skopeo(1) section "IMAGE NAMES" for the expected format
 	flags.StringSliceVar(&opts.encryptionKeys, "encryption-key", []string{}, "*Experimental* key with the encryption protocol to use needed to encrypt the image (e.g. jwe:/path/to/key.pem)")
 	flags.IntSliceVar(&opts.encryptLayer, "encrypt-layer", []int{}, "*Experimental* the 0-indexed layer indices, with support for negative indexing (e.g. 0 is the first layer, -1 is the last layer)")
 	flags.StringSliceVar(&opts.decryptionKeys, "decryption-key", []string{}, "*Experimental* key needed to decrypt the image")
+	flags.UintVar(&opts.maxParallelDownloads, "max-parallel-downloads", 6, "Maximum number of parallel downloads")
```

Comment:
The “downloads” name is not really accurate; `skopeo copy` could have well been used for a “push” or a registry--to-registry-copy.

(Yes, the `copy.Options` field uses that naming. That was a mistake.)

`containers.conf` uses `image_parallel_copies`; that could probably be also improved upon, but at least it’s better in this respect, and the consistency seems attractive to me.